src/screens/LegalRights/index.tsx

```diff
@@ -1,5 +1,163 @@
-import { Text } from 'react-native';
+import { useState } from 'react';
+import { Button, Image, Pressable, ScrollView, Text, View } from 'react-native';
+import placeholderPoster from '@/assets/images/placeholder.png';
+import { styles } from './styles';
 
-export default function LegalRights() {
-  return <Text>LEGAL RIGHTS!!</Text>;
+export default function LegalRights({ navigation }: { navigation: any }) {
+  function rick_rolls() {
```

Comment:
just for consistency, let's make these into anonymous functions! Something like `const rickRolls = () => {}`. Also just be sure to use camelCase, as it's convention